server/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+
+const app = express();
+const port = 3003;
+const db = require('../database/index.js');
+
+let allowCrossDomain = function(req, res, next) {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header('Access-Control-Allow-Headers', '*');
+  next();
+ }
+ app.use(allowCrossDomain);
+
+app.get('/api/images/:productId', (req, res) => {
+  const { productId } = req.params;
+  console.log('proid: ', productId);
+
+  db.findOne({ productId: productId }, (err, results) => {
```

Comment:
nit: Not sure if you have the Airbnb linter, but it is possible to de-structure productId to {productId}

Reply:
I originally was using de-structuring, but changed it to be like this when hunting a bug. I will change it back to be de-structured.